provider_shopper/test/cart_widget_test.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+import 'package:provider_shopper/models/cart.dart';
+import 'package:provider_shopper/models/catalog.dart';
+import 'package:provider_shopper/screens/cart.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  Widget cartScreen() => MultiProvider(
```

Comment:
Typically we don't created nested functions like this, so it'd be better to move this into global scope in the file. I'd recommend renaming it to "createCartScreen", as well.